controllers/storagecluster/storagecluster_controller.go

```diff
@@ -113,12 +121,31 @@ func (r *StorageClusterReconciler) SetupWithManager(mgr ctrl.Manager) error {
 		},
 	}
 
+	enqueueStorageClusterRequest := handler.EnqueueRequestsFromMapFunc(
```

Comment:
I think it will be better if the name of the enqueue command will describe the fact that this request is enqueued from a label.
```suggestion
	enqueueStorageClusterRequestFromLabel := handler.EnqueueRequestsFromMapFunc(
```